validation/validator.go

```diff
@@ -261,6 +264,93 @@ func (v *successValidator) ValidateParameterBase64(prmName string, value *string
 	return v
 }
 
+func (v *successValidator) ValidateParameterMaxSize(prmName string, value *[]byte, maxSize int, mandatory bool) Validator {
```

Comment:
You could have used ValidateParameterIntBetween... why would we limit the maximum and not the minimum? For a logo, or favicon, I don't think that a 10 bytes value would be acceptable